arch/platform/srf06-cc26xx/platform.c

```diff
@@ -209,7 +209,13 @@ platform_init_stage_three()
           ti_lib_chipinfo_chip_family_is_cc13xx() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_ble() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_proprietary() == true ? "Yes" : "No");
-  LOG_INFO(" RF: Channel %d, PANID 0x%04X\n", chan, pan);
+  LOG_INFO(" RF: Channel %d", chan);
+
+  if(NETSTACK_RADIO.get_value(RADIO_PARAM_PAN_ID, &pan) == RADIO_RESULT_OK) {
+    LOG_INFO(", PANID 0x%04X", pan);
```

Comment:
I still think you want `LOG_INFO_` here instead (note the final `_`)

Reply:
Still not used to this... So I need `INFO_` here as well as in the final `\n`

Reply:
Yes, and actually also the "RF: channel one". You need it whenever continuing a previous line of log. Else a prefix will added every time

Reply:
Yep. The "RF: Channel" is at the start of a line.